src/components/WwwFrame/TheTipMessage.vue

```diff
@@ -103,22 +103,26 @@ export default {
 	},
 	computed: {
 		currentMessage() {
+			// return tip message if there are no flash messages
+			if (!this.flashMessages?.length) {
+				return this.tipMessage;
+			}
 			// return tip message if it's level is higher than first flash message, otherwise return first flash message
 			return this.tipMessage?.type && compareLevels(this.tipMessage.type, this.flashMessages[0]?.messageType)
 				? this.tipMessage
 				: this.flashMessages[0];
 		},
 		persist() {
 			const type = this.currentMessage?.type ?? this.currentMessage?.messageType ?? '';
-			return !!this.currentMessage?.persist || type === TIP || type === WARNING;
+			return !!this.currentMessage?.persist || type === ERROR || type === WARNING;
 		},
 	},
 	methods: {
 		showCurrentMessage() {
 			const message = this.currentMessage?.message ?? '';
 			const safeMessage = DOMPurify.sanitize(message, { ALLOWED_TAGS: ['b', 'i', 'em', 'strong', 'a'] });
 			const type = this.currentMessage?.type ?? this.currentMessage?.messageType ?? '';
-			this.$refs.tip?.show(safeMessage, type, this.persist);
+			this.$refs.tip?.show(safeMessage, type === TIP ? '' : type, this.persist);
```

Comment:
KvToast doesn't handle the 'tip' type yet, so passing an empty string to get the default instead for now.